docs/core/common/assets/js/api/core/ui-states.md

```diff
@@ -0,0 +1,180 @@
+## API - `$e.uiStates`
+*  **Description**: `$e.uiStates` API is a UI state manager that allows you to create custom UI states for your components.
+   We've added this API since we needed a way to reflect the component's state in the UI itself and react accordingly
```

Comment:
Please remove all `we` / `you` / `us` etc. in the docs.

```suggestion
   It reflects the component's state in the UI itself and allows to react accordingly
```

Reply:
Done